webpack.config.js

```diff
@@ -48,4 +48,12 @@ webpackConfig.resolve.modules = [
 	'node_modules'
 ]
 
+// Fix Buffer issues
+webpackConfig.plugins.push(new webpack.ProvidePlugin({
+	Buffer: ['buffer', 'Buffer'],
+}))
+webpackConfig.resolve.fallback = {
+	buffer: require.resolve('buffer'),
+}
```

Comment:
Should this be upstreamed?

Reply:
No clue, but it's required somehow
ref https://github.com/nextcloud-libraries/webpack-vue-config/issues/450#issuecomment-1674291089